webpack/JobInvocationDetail/CheckboxesActions.js

```diff
@@ -76,10 +91,82 @@ export const CheckboxesActions = ({ selectedIds, failedCount, jobID }) => {
     setIsModalOpen(true);
   };
 
+  const filterQuery =
+    filter && filter !== 'all_statuses'
+      ? ` and job_invocation.result = ${filter}`
+      : '';
+  const combinedQuery = `${bulkParams}${filterQuery}`;
+
+  const handleTaskAction = action => {
+    dispatch(
+      addToast({
+        key: `cancel-job-info`,
+        type: 'info',
+        message: sprintf(__('Trying to %s the task for the host'), action),
+      })
+    );
+
+    dispatch(
+      APIActions.post({
+        url: `/api/v2/job_invocations/${jobID}/cancel`,
+        key: 'CANCEL_JOB_TASKS_BY_SEARCH',
+        params: {
+          search: combinedQuery,
+          force: action !== 'cancel',
+        },
+        errorToast: ({ response }) => response?.data?.error,
+        successToast: ({ response }) => {
```

Comment:
```suggestion
        successToast: ( response ) => {
```
response is not a key